addons/MountMuzzle/constants.lua

```diff
@@ -62,6 +62,7 @@ music = {
         mount = 84,
         chocobo = 212,
         zone = 0,
+        custom = 9998
```

Comment:
I assume these are song IDs and you uses `9998` just to make sure it doesn't exist, and because `9999` was already taken? If so, would it be an issue if they both uses `9999`? In general, I'd prefer clearly separate numbers, like negative numbers, or ideally just leaving them `nil`, if that works with the rest of the code.

Reply:
I don't think there'd be any issue with making it also 9999. I will test it out.